src/main/java/eci/ieti/leafOut/backEnd/model/Park.java

```diff
@@ -1,69 +1,52 @@
 package eci.ieti.leafout.backend.model;
 
 
+import lombok.*;
+
 import java.util.List;
 
-public class Park {
+/**
+ * This class represents a park
+ *
+ * @author Juan Pablo Ospina Henao
+ * @since 0.0.1
+ */
+
 
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Park {
+    /**
+     *  id number of a park
```

Comment:
As far as the name of the attribute is meaningful enough, you don't need to add this documentation